lib/Keys.ts

```diff
@@ -1,5 +1,5 @@
 
 import { ECDSAPrivateKey, ECDSAPublicKey } from './ECDSA';
 
-export type PublicKey = ECDSAPublicKey;
-export type PrivateKey = ECDSAPrivateKey;
+export interface PublicKey extends ECDSAPublicKey {}
+export interface PrivateKey extends ECDSAPrivateKey {}
```

Comment:
Changing from `type` to `interface` avoids the inlined `import` in the compiled code for `PublicKey` in `DeonData.d.ts`.

It's contraband to this PR.